src/Analyzers/CSharp/CodeFixes/MakeMethodAsynchronous/CSharpMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -177,7 +180,21 @@ private static bool IsIEnumerator(ITypeSymbol returnType, KnownTaskTypes knownTy
     private static (SyntaxTokenList newModifiers, TypeSyntax newReturnType) AddAsyncModifierWithCorrectedTrivia(SyntaxTokenList modifiers, TypeSyntax returnType)
     {
         if (modifiers.Any())
-            return (modifiers.Add(s_asyncKeywordWithSpace), returnType);
+        {
+            // 'partial' modifier must say at the end of the modifiers arrays.
```

Comment:
I think `await` completion that I implemented may have the exact same bug. The bug is more critical for IntelliSense I think.

Reply:
https://github.com/dotnet/roslyn/blob/5213c18c0d2d74b688b745a96f7e76e67cb961a0/src/Features/CSharp/Portable/Completion/CompletionProviders/AwaitCompletionProvider.cs#L40